cli/CHANGELOG.md

```diff
@@ -1,17 +1,23 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
-## 12.10.1
 
-_Released 04/25/2023 (PENDING)_
+## 12.11.0
```

Comment:
This needs to be on line 2

Reply:
removed in f0247bde3af51943378cc18041631d0a630b92d3